Assets/Scripts/Gameplay/PlayerEnemyCollision.cs

```diff
@@ -15,6 +15,7 @@ public class PlayerEnemyCollision : Simulation.Event<PlayerEnemyCollision>
     {
         public EnemyController enemy;
         public PlayerController player;
+        public HealthController healthCon;
```

Comment:
no point having this here cos it's done below and never persisted

Reply:
ok rectifed